app/assets/stylesheets/_entries.scss

```diff
@@ -7,43 +7,40 @@
     @include pad(5%)
   }
 
-  #new_entry, .edit_entry {
-
-    .button {
-      width: 100%;
-    }
+  .button {
+    width: 100%;
+  }
 
-    .input {
-      margin: 10px 0;
+  .input {
+    margin: 10px 0;
 
-      input {
-        resize: none;
-        -webkit-font-smoothing: antialiased;
-        font-smoothing: antialiased;
-      }
+    input {
+      resize: none;
+      -webkit-font-smoothing: antialiased;
+      font-smoothing: antialiased;
     }
+  }
 
-    .selectize-dropdown, .selectize-input, .selectize-input input {
-      font-size: $base-font-size;
-      font-family: 'Open Sans';
-    }
+  .selectize-dropdown, .selectize-input, .selectize-input input {
```

Comment:
Selector should have depth of applicability no greater than 2, but was 3<br>Each selector in a comma sequence should be on its own line